src/lib.rs

```diff
@@ -286,7 +286,7 @@ impl fmt::Display for Error {
                         let crate_name =
                             std::env::var("CARGO_PKG_NAME").unwrap_or_else(|_| "sys".to_owned());
                         let instructions = if cfg!(target_os = "macos") || cfg!(target_os = "ios") {
```

Comment:
My only comment here is that this message should only be printed when targeting macos, not ios, since homebrew doesn't target ios (AFAIK).

Reply:
indeed, homebrew does not support ios